Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -563,7 +563,7 @@ public IEnumerable<EntityUid> GetColliding(EntityUid uid, PhysicsComponent? phys
         var tileRef = _mapSystem.GetTileRef(xform.GridUid.Value, mapGridComp, xform.Coordinates);
 
         _doorIntersecting.Clear();
-        _entityLookup.GetLocalEntitiesIntersecting(xform.GridUid.Value, tileRef.GridIndices, _doorIntersecting, gridComp: mapGridComp, flags: (LookupFlags.All & ~LookupFlags.Sensors));
+        _entityLookup.GetLocalEntitiesIntersecting(xform.GridUid.Value, tileRef.GridIndices, _doorIntersecting, -0.1f, gridComp: mapGridComp, flags: (LookupFlags.All & ~LookupFlags.Sensors));
```

Comment:
Replaces a default of -0.04, which for some reason is now insufficient.

I looked  into not doing this in general but this is messy physics nonsense.